packages/components/src/dialog/src/Dialog.css

```diff
@@ -17,15 +17,27 @@
     align-items: flex-start;
 }
 
+.o-ui-dialog-wrapper-fullscreen {
+    padding: 0;
+}
+
 /* DIALOG */
 .o-ui-dialog {
     position: relative;
     background-color: var(--o-ui-bg-alias-default);
     border-radius: var(--o-ui-br-rounded);
     display: grid;
     align-items: stretch;
-    grid-template-areas: "image aside";
-    grid-template-columns: min-content auto;
+    grid-template-areas: "image image" "aside aside";
+    grid-template-rows: min-content auto;
+}
+
+@media screen and (min-width: 768px) {
```

Comment:
I don't like the idea of duplicating the breakpoints values.

Maybe `createThemeVars` could render CSS vars for the breakpoint values then you could use the values here?

Those vars could also be useful for our apps.

Reply:
> I don't like the idea of duplicating the breakpoints values.
> 
> Maybe `createThemeVars` could render CSS vars for the breakpoint values then you could use the values here?
> 
> Those vars could also be useful for our apps.

not a bad idea!

Reply:
:(